packages/react/src/contexts/talentLayerContext.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import React, { ContextType } from 'react';
+import { createContext, ReactNode, useEffect, useMemo, useState } from 'react';
+import { useAccount } from 'wagmi';
+import { IAccount, IUser, NetworkEnum } from '../types';
+import { TalentLayerClient } from '@talentlayer/client';
+
+interface TalentLayerProviderProps {
+  children: ReactNode;
+  config: ConstructorParameters<typeof TalentLayerClient>[0] & {
```

Comment:
anyway to simplify the config here?
Its a little tough to understand what the type is.